requirements.txt

```diff
@@ -1,8 +1,7 @@
 canonicalwebteam.flask-base==2.0.0
 canonicalwebteam.blog==6.4.2
 canonicalwebteam.http==1.0.4
-canonicalwebteam.image-template==1.3.1
-canonicalwebteam.templatefinder==1.0.0
```

Comment:
@petesfrench Why are we removing template finder?

Reply:
That is my mistake, I took it out as I was trying to isolate the source of a bug and forgot to put it back in.